### DISCUSSION TITLE
Reference to eslint-plugin-remix-react-routes

### DISCUSSION THREAD

**Discussion by ryanwoodcox** [epicweb-dev/epic-stack#757] [Q&A] [answered]
After the adoption of @epic-web/config it does not appear that the epic-stack references the eslint-plugin-remix-react-routes package.
Additionally, I do not see @epic-web/config referencing this package. 

1. Should this be package be uninstalled from the epic-stack?
2. Was this package's plugin and its respective rules no longer valuable?

**Comment by kentcdodds** [comment]
I found it wasn't working and removed it rather than work out why. We'll be getting type safe routes in the future so I'm not so motivated to get it fixed myself.

**Reply by ryanwoodcox** [reply]
Thanks for the additional information.
To revisit question 1,  it sounds like eslint-plugin-remix-react-routes should be uninstalled from epic-stack?

**Reply by kentcdodds** [reply]
Yes, it was uninstalled. But you can keep it in your project if you're happy with it.

**Reply by ryanwoodcox** [reply]
I'm asking because I see it still in the package.json
https://github.com/epicweb-dev/epic-stack/blob/5fa0dd796f02b8702fea1eef3e853984b4d532fa/package.json#L79

**Reply by kentcdodds** [reply]
Oh whoops! I only checked the devDeps which is where it should've been in the first place. I'd welcome a PR to remove it

**Comment by kentcdodds** [comment, marked as answer]
It's been removed
